### PR TITLE
feat: add guideline-themes package

### PR DESCRIPTION
**Pull request by anxobotana**

### CODE REVIEW

File: packages/guideline-themes-settings/package.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "test": "vitest run --silent",
+        "test:coverage": "vitest --coverage",
+        "test:watch": "vitest",
+        "test:ui": "vitest --ui",
+        "typecheck": "tsc --noEmit"
+    },
+    "peerDependencies": {
+        "@frontify/app-bridge": "^3.0.0 || ^4.0.0-alpha",
```
Comment (ragi96) [inline packages/guideline-themes-settings/package.json:44]: Imo you don't need it as peerDep

File: .github/workflows/guideline-themes-continuous-integration.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            - name: Get pnpm store directory
+              id: pnpm-cache
+              run: echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+
+            - name: Setup pnpm cache
+              uses: actions/cache@v3
+              with:
+                  path: |
+                      ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+                  key: pnpm-with-${{ matrix.app-bridge-version }}-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
```
Comment (ragi96) [inline .github/workflows/guideline-themes-continuous-integration.yml:77]: No need for matrix as it doesn't need to be a peerDep

**Review by ragi96 — COMMENTED** [review]
Just my 5 cents as i saw it

File: packages/guideline-themes-settings/package.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
+{
+    "name": "@frontify/guideline-themes-settings",
```
Comment (SamuelAlev) [inline packages/guideline-themes-settings/package.json:2]: Isn't what Mike proposed? so we merge `app-bridge-theme` and `guideline-themes` into a single package? 
```suggestion
    "name": "@frontify/theme-brand-sdk",
```

File: packages/guideline-themes-settings/CHANGELOG.md
Comment (SamuelAlev) [inline packages/guideline-themes-settings/CHANGELOG.md:1]: File can be deleted as changeset will create it

File: .github/workflows/guideline-themes-continuous-integration.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            - name: Use Node.js
+              uses: actions/setup-node@v4
+              with:
+                  node-version-file: ".nvmrc"
+                  cache: "pnpm"
+
+            - name: Use pnpm
+              uses: pnpm/action-setup@v3
+              with:
+                  run_install: false
```
Comment (SamuelAlev) [inline .github/workflows/guideline-themes-continuous-integration.yml:67]: They need to be swapped
```suggestion
            - name: Use pnpm
              uses: pnpm/action-setup@v3
              with:
                  run_install: false
                  
            - name: Use Node.js
              uses: actions/setup-node@v4
              with:
                  node-version-file: ".nvmrc"
                  cache: "pnpm"
```

File: packages/guideline-themes-settings/package.json [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        "typecheck": "tsc --noEmit"
+    },
+    "peerDependencies": {
+        "react": "^18",
+        "react-dom": "^18"
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.9.20"
+    },
+    "devDependencies": {
```
Comment (mike85) [inline packages/guideline-themes-settings/package.json:50]: Please remove all the dependencies that are not needed.

File: packages/guideline-themes-settings/postcss.config.cjs
Comment (mike85) [inline packages/guideline-themes-settings/postcss.config.cjs:1]: This file is not needed.

File: packages/guideline-themes-settings/src/theme.ts
Comment (mike85) [inline packages/guideline-themes-settings/src/theme.ts:1]: We might put this into `index.ts` as they do in `guideline-blocks-settings`. Wdyt?

File: packages/guideline-themes-settings/tailwind.config.ts
Comment (mike85) [inline packages/guideline-themes-settings/tailwind.config.ts:1]: We don’t need tailwind in our package.

File: packages/guideline-themes-settings/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react-jsx",
+        "strict": true,
+        "noUncheckedIndexedAccess": false,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noFallthroughCasesInSwitch": true,
+        "baseUrl": ".",
+        "types": ["@testing-library/jest-dom", "cypress", "cypress-real-events", "node"]
```
Comment (mike85) [inline packages/guideline-themes-settings/tsconfig.json:20]: Cypress is not needed in our package.

File: packages/guideline-themes-settings/tsconfig.node.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    "compilerOptions": {
+        "composite": true,
+        "module": "ESNext",
+        "moduleResolution": "Node",
+        "allowSyntheticDefaultImports": true,
+        "resolveJsonModule": true,
+        "lib": ["ESNext"],
+        "target": "ESNext"
+    },
+    "include": [".eslintrc.cjs", "postcss.config.cjs", "tailwind.config.ts", "vite.config.ts", "package.json"]
```
Comment (mike85) [inline packages/guideline-themes-settings/tsconfig.node.json:11]:
```suggestion
    "include": [".eslintrc.cjs", "vite.config.ts", "package.json"]
```

File: packages/guideline-themes-settings/vite.config.ts [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+const peerDependencies = Object.keys(peerDependenciesMap);
+
+export const globals = {
+    react: 'React',
+    'react-dom': 'ReactDOM',
+    'react-dom/client': 'ReactDOM',
+};
+
+const assetFileNames = (chunkInfo: PreRenderedAsset): string => {
+    if (chunkInfo.name === 'style.css') {
```
Comment (mike85) [inline packages/guideline-themes-settings/vite.config.ts:22]: We don’t use a style.css

**Review by mike85 — CHANGES_REQUESTED** [review]

File: packages/guideline-themes-settings/package.json [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        "test:coverage": "vitest --coverage",
+        "test:watch": "vitest",
+        "test:ui": "vitest --ui",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.9.20"
+    },
+    "devDependencies": {
+        "@frontify/app-bridge-theme": "0.0.0-alpha.17",
```
Comment (SamuelAlev) [inline packages/guideline-themes-settings/package.json:44]: If [this comment](https://github.com/Frontify/brand-sdk/pull/1087/files#r1735928135) isn't applicable, then we need to do this and also move it to `dependencies`:
```suggestion
        "@frontify/app-bridge-theme": "workspace:^",
```

**Review by mike85 — APPROVED** [review]
LGTM

File: packages/guideline-themes-settings/.eslintrc.cjs [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+                '@typescript-eslint/no-misused-promises': 'warn',
+                '@typescript-eslint/no-unsafe-argument': 'warn',
+                '@typescript-eslint/no-unsafe-assignment': 'warn',
+                '@typescript-eslint/no-unsafe-call': 'warn',
+                '@typescript-eslint/no-unsafe-member-access': 'warn',
+                '@typescript-eslint/no-unsafe-return': 'warn',
+                '@typescript-eslint/require-await': 'warn',
+                '@typescript-eslint/await-thenable': 'warn',
+                '@typescript-eslint/no-unsafe-enum-comparison': 'warn',
+                '@typescript-eslint/restrict-plus-operands': 'warn',
```
Comment (ragi96) [inline packages/guideline-themes-settings/.eslintrc.cjs:47]: Probably you can be more strict here, and don't need these overwrites

File: packages/guideline-themes-settings/package.json (hunk 3 above)
Comment (SamuelAlev) [reply packages/guideline-themes-settings/package.json:2]: Sure, let's align with how platform app is then (without `-settings`), blocks team has also in their backlog the task to remove `-settings` to just keep `@frontify/guideline-blocks`.

```suggestion
    "name": "@frontify/guideline-themes",
```
(+ folder rename)

**Pull request merged**